modules/ROOT/pages/manage-versions-instances-concept.adoc

```diff
@@ -3,9 +3,19 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-After you create an API in Anypoint Platform, you configure the version details for the API and then publish it to Anypoint Exchange (Exchange). The API version resides in Exchange, and not in API Manager. However, you then manage this API version from API Manager.
+After an API is created, you can configure version details for the API and publish them to Anypoint Exchange (Exchange). The API asset version resides in Exchange, and not in API Manager. However, you manage the version of an API instance from API Manager. You can create multiple instances and expand the version information to view details about them.
 
-After setting up an API version for management, you repeat the process to create another instance. Attempting to configure the same version creates another instance of the version. You can create multiple instances and expand the version information to view details about them.
```

Comment:
I would not remove this part, maybe we can rephrase and clarify it a bit. But here is where we say that if they create a new API Instance linked to the same Exchange API Asset version, a completely new API Instance will be created in API Manager.

> You can create multiple instances and expand the version information to view details about them.

I'm not sure what does "expand the version information to view details about them" mean 🤔

Reply:
This section still really confuses me as well. I feel like it's clear that creating a new API just creates a new instance then you manage this instance, but I don't know. Let me know what you think about this in the new one.